Example/Authenticator/View Controllers/Connection/ConnectionsViewController.swift

```diff
@@ -217,47 +217,48 @@ extension ConnectionsViewController {
 }
 
 // MARK: - Actions
+// TODO: Refactor
 private extension ConnectionsViewController {
     func showActionSheet(at indexPath: IndexPath) {
-        let actionSheet = CustomActionSheetViewController()
-
-        let reconnectAction: Action = { [weak self] in actionSheet.dismissActionSheetWithCompletion {
-                guard let connectionId = self?.viewControllerViewModel.connectionId(at: indexPath) else { return }
-
-                self?.reconnect(connectionId)
-            }
-        }
-
-        let contactSupportAction: Action = { [weak self] in actionSheet.dismissActionSheetWithCompletion {
-                guard let connectionViewModel = self?.viewControllerViewModel.cellViewModel(at: indexPath) else { return }
-
-                self?.showSupportMailComposer(withEmail: connectionViewModel.supportEmail)
-            }
-        }
-
-        let renameAction: Action = { [weak self] in actionSheet.dismissActionSheetWithCompletion {
-                guard let connectionId = self?.viewControllerViewModel.connectionId(at: indexPath) else { return }
-
-                self?.rename(connectionId)
-            }
-        }
-
-        let deleteAction: Action = { [weak self] in actionSheet.dismissActionSheetWithCompletion {
-                self?.viewControllerViewModel.remove(at: indexPath)
-            }
-        }
-
-        var actionsArray: [(actionSheetItem: ActionSheetAction, action: Action)] = [
-            (.rename, renameAction),
-            (.support, contactSupportAction),
-            (.delete, deleteAction)
-        ]
-
-        if viewControllerViewModel.cellViewModel(at: indexPath).status == ConnectionStatus.inactive.rawValue {
-            actionsArray.insert((.reconnect, reconnectAction), at: 0)
-        }
-
-        actionSheet.actions = ConnectionActionSheetBuilder.createActions(from: actionsArray)
+//        let actionSheet = CustomActionSheetViewController()
```

Comment:
I think, that you can surely remove all this code and everything, that depends on `CustomActionSheetViewController`. There will be other logic.